crates/cranelift/src/compiler/component.rs

```diff
@@ -254,6 +516,84 @@ impl Compiler {
         })
     }
 
+    fn load_libcall(
+        &self,
+        builder: &mut FunctionBuilder<'_>,
+        offsets: &VMComponentOffsets<u8>,
+        vmctx: ir::Value,
+        offset: u32,
```

Comment:
True! There's a newtype for core wasm libcalls but I was a bit hesitant to duplicate all the macro machinery for that as well since the libcalls already duplicate other macro machinery.